ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-presubmits.yaml

```diff
@@ -494,6 +494,75 @@ presubmits:
           - configMap:
               name: cluster-profile-gcp
     trigger: ((?m)^/test( all| gcp-major-upgrade),?(\s+|$))
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - release-3.10
+    context: ci/prow/gcp-major-upgrade
+    decorate: true
+    name: pull-ci-openshift-openshift-ansible-310-gcp-major-upgrade
+    rerun_command: /test gcp-major-upgrade
+    skip_cloning: true
+    spec:
+      containers:
+      - args:
+        - |
+          export PREVIOUS_RPM_REPO="$( curl -q "${PREVIOUS_RPM_REPO_BASEREF}" 2>/dev/null)"
+          ci-operator \
+            --artifact-dir=$(ARTIFACTS) \
+            --secret-dir=/usr/local/e2e-gcp-cluster-profile \
+            --template=/usr/local/e2e-gcp-major-upgrade-310 \
+            --target=e2e-gcp-major-upgrade-310 \
+            --give-pr-author-access-to-namespace=true
+        command:
+        - /bin/bash
+        - -c
+        env:
+        - name: TEST_FOCUS
+          value: Suite:openshift/conformance/parallel
+        - name: JOB_NAME_SAFE
+          value: e2e-gcp-major-upgrade-310
+        - name: CLUSTER_TYPE
+          value: gcp
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: release-3.10.yaml
+              name: ci-operator-openshift-openshift-ansible
+        - name: RPM_REPO
+          value: https://rpms.svc.ci.openshift.org/openshift-origin-v3.10/
+        - name: PREVIOUS_RPM_REPO_BASEREF
+          value: https://storage.googleapis.com/origin-ci-test/releases/openshift/origin/release-3.9/.latest-rpms
```

Comment:
@smarterclayton: are there plans to add 3.9 and 3.10 rpm servers as was done in #1407?

Reply:
This was recommended in https://github.com/openshift/release/pull/1298#issuecomment-416658036 btw. I don't think its worth setting up a server for openshift-ansible only. Also we're not planning to backport this further anyway